enzyme/Enzyme/AdjointGenerator.h

```diff
@@ -248,16 +246,37 @@ class AdjointGenerator
         Value *orig_op1 = FPMO->getOperand(0);
```

Comment:
This also appears to be missing an `eraseIfUnused(*FPMO);`, mind adding it here for correctness.

Reply:
wouldn't line 244/242 already take care of this? `eraseIfUnused(inst);`

Reply:
Yes it would! The diff cut it off and I was a bit concerned.

Reply:
allright 😄